### ISSUE TITLE
DLL: String as Iterable on initialization

### ISSUE THREAD

**Issue by CCallahanIV** [wenima/data-structures#34]
Problem with initialization - strings will be treated as an iterable and a Dbl Node will be created for each character in the string.

Change to test for string.

**Comment by wenima** [comment]
yeah, I actually asked Nic about. Technically speaking, the code is working fine, a string is an iterable. If a string should be the value of a single node, it needs to be pased in as an element of an iterable like a tuple or a list. 
This would be a code change as well as a new test and would expand on the requirement by making assumptions I guess but I did the same for an interger and wasted time when it wasn't needed so I can go either way. Good thinking though!!

**Comment by CCallahanIV** [comment]
I thought more about this.  I'm beginning to change my mind.  It makes sense that if you're initializing a DLL with a string, you'd likely WANT it be split into different nodes.

I'm going to leave it as is, we can discuss this on Monday if there's any more doubt.
